### DISCUSSION TITLE
语言/Language

### DISCUSSION THREAD

**Discussion by Nokia-C** [lencx/ChatGPT#517] [Ideas]
Hi board owner, I am a Chinese mainland user, please ask the next update, you can add Chinese language? The interface will be completely Chinese, thank you for your hard work~（你好板主，我是中国大陆用记，请问下次更新，您能添加中文语言吗？将界面完全汉化，感谢辛苦你了~)

**Comment by haifenggxbl** [comment]
支持！！！！！！！！！！！！！！！！！！

**Comment by lencx** [comment]
汉化需要对整个项目结构进行调整，所以暂时不会在当前项目中提供支持。我会在新项目 https://github.com/lencx/nofwl 中支持国际化，以及更加复杂的功能（可以理解为 ChatGPT 只是新项目的一个子模块）。

**Comment by LinaZhao9911** [comment]
你好版主，我发现中文圣经经文是chatGPT在线翻译的，可以请你帮忙把中文圣经原版（代码：cunpss from bible.com）喂给chatGPT 吗？谢谢你！
